src/components/Dialogs/ConfirmationDialog.vue

```diff
@@ -10,11 +10,22 @@
       </p>
 
       <div class="flex flex-col s:flex-row gap-4">
-        <the-button :regularButton="true" :isInForm="true" class="red-button">
+        <the-button
+          @click="submit"
+          :regularButton="true"
+          :isInForm="true"
+          class="red-button"
+        >
           Delete
         </the-button>
 
-        <the-button :regularButton="true" :isInForm="true" class="white-button">
+        <the-button
+          @click="$emit('close-dialog')"
+          :regularButton="true"
+          :isInForm="true"
+          type="button"
```

Comment:
is this attr needed?

Reply:
If you mean the `type="button"` attribute then yeah - it makes a button to not submit the form, according to this [stackoverflow thread](https://stackoverflow.com/questions/3314989/can-i-make-a-button-not-submit-a-form).

Reply:
You don't rely on native form submission AFAIR so this would be obsolete.

Reply:
I'm not sure what you mean by me not relying on native form submission, but the thing is that I have those all add/edit/delete forms and each of them has **two** buttons (I can't attach the screen right now but I sent you screens of those so many times so I believe you remember how they look). One of them is for actually submitting the form but the other one is the "cancel" or "add new input" button.

And now, when I click let's say this cancel button, the submission takes place and the data are sent to firebase. Which, obviously, isn't the desired behavior since I don't want to send any data when I click the cancel button. And adding this type attribute, apparently, prevents the form submission when clicking the respective button which is exactly what I want to achieve here.